app/views/superpowers/show.html.erb

```diff
@@ -1,3 +1,6 @@
+<hr />
+<h1>ABOVE IS REACT</h1>
```

Comment:
This shouldn't be left in the PR - ideally any PRs should be entirely ready to merge into our Production site.